bootstrap.sh

```diff
@@ -269,8 +269,7 @@ function release {
     boxes
     aztec-up
     playground
-    # Blocking release.
-    # docs
+    docs
```

Comment:
this enables docs builds on the CI (it calls the `""|"fast"|"full"` command in the docs folder